### ISSUE TITLE
Avg Loss can't down from 35

### ISSUE THREAD

**Issue by aegisrox** [pjreddie/darknet#793]
Hi Nice to meet you. I have a problem training yolo, and would be awesome some assistence.
I'm training yolo with 41 Classes and after 5000 the avg never goes down under 40-36.
I tried lowering the learning rate and growing up the iterations but nothing changes.

Any idea what could i try?

PS: I'm using the the yolo_voc.2.0.cfg by the way

**Comment by khiemntu** [comment]
I think you should try yolov3, maybe it help your case

**Comment by wenku1992abc** [comment]
@khiemntu Are you sure  yoloV3 convergent faster than yoloV2 ? Why my yoloV3 convergent lower than yoloV2?

**Comment by khiemntu** [comment]
@wenku1992abc I have tested on yolov3 and yolov2 on the same dataset.

**Comment by aegisrox** [comment]
Thanks for the answers. I trained using yolov3.cfg with the same dataset but i have another problem.

Now the loss is pretty lower but at the demo detector i'm getting none results.

During the training I get some NaN values using this configuration. Any ideas what could it happen?

**Comment by khiemntu** [comment]
you got none result when detect ? Are you change batch=1 and sub=1 in cfg file ? NaN is not a problem when trainning. Only you got  all nan, that is training goes wrong

**Comment by aegisrox** [comment]
Thanks again @khiemntu  now i have batch=32 and subdivision=16. This is correct?

**Comment by khiemntu** [comment]
when you test, you need set batch=1 and sub=1
